smoke_tests.sh

```diff
@@ -1,4 +1,15 @@
 #!/usr/bin/env bash
 set -ex
 
-[ $(curl --write-out %{http_code} --silent --output /dev/null $(hostname)/medium-articles) == 200 ]
+host=$(hostname)
+# host="localhost:8088"
+
+successResponse=$(curl --write-out %{http_code} --silent --output /dev/null ${host}/medium-articles)
+
+if [ "$successResponse" == 200 ]
+then
+    exit 0
+else
+   curl ${host}/medium-articles
+   exit 1
+fi
```

Comment:
It it's helpful to debug when it breaks, let's leave it there. Could try to capture the original output of the failed response by changing the `--output` option to a `/tmp/...` file, rather than repeating the request and possibly get a different result.